celestia.go

```diff
@@ -110,14 +136,29 @@ func (c *CelestiaDA) Validate(ids []da.ID, daProofs []da.Proof) ([]bool, error)
 		proofs = append(proofs, proof)
 	}
 	for i, id := range ids {
-		// TODO: extract commitment from ID
-		isIncluded, err := c.client.Blob.Included(c.ctx, c.height, share.Namespace(c.namespace), proofs[i], blob.Commitment(id))
-		if err != nil {
-			return nil, err
-		}
+		height, commitment := splitID(id)
+		// TODO(tzdybal): for some reason, if proof doesn't match commitment, API returns (false, "blob: invalid proof")
+		//    but analysis of the code in celestia-node implies this should never happen - maybe it's caused by openrpc?
+		//    there is no way of gently handling errors here, but returned value is fine for us
+		isIncluded, err := c.client.Blob.Included(c.ctx, height, c.namespace, proofs[i], commitment)
+		log.Println("Included: ", isIncluded, err)
```

Comment:
Should match previous log messages:

> log.Println("successfully submitted blobs", "height", height)

for consistency.

Reply:
Actually it's just a debug output I forgot to remove.